docs/en/get_started.md

```diff
@@ -63,6 +63,16 @@ Case b: If you use mmsegmentation as a dependency or third-party package, instal
 pip install mmsegmentation
 ```
 
+**Step 2.** Install extra dependencies for Albumentations
```

Comment:
```suggestion
```
As albumentations is not tightly coupled with mmseg, I think it would be better that keep the installation of mmseg simple here.